package.json

```diff
@@ -4,15 +4,18 @@
   "main": "index.js",
   "version": "0.0.0-development",
   "scripts": {
+    "build": "babel ./index.js -d build -s",
     "commit": "git-cz",
     "check-coverage": "istanbul check-coverage --statements 100 --branches 90 --functions 100 --lines 100",
+    "install": "babel ./index.js -d build -s",
     "report-coverage": "cat ./coverage/lcov.info | codecov",
     "test:watch": "istanbul cover -x *.test.js _mocha -- -R spec -w index.test.js",
     "test": "istanbul cover -x *.test.js _mocha -- -R spec index.test.js",
-    "semantic-release": "semantic-release pre && npm publish && semantic-release post"
+    "semantic-release": "semantic-release pre && npm publish && semantic-release post",
+    "watch": "babel -w ./index.js -d build -s"
```

Comment:
I would prefer this to be called `build:watch` to follow the convention we have with `test`.